src/utils/MediaQueries.js

```diff
@@ -1,10 +1,12 @@
 const size = {
   mobileS: '360px',
+  onlyMobileS: '1023px',
   laptop: '1024px',
   laptopL: '1440px',
 };
 
 const device = {
+  onlyMobileS: `(max-width: ${size.onlyMobileS})`,
```

Comment:
onlyMobileS property in device object is OK. But not "size.onlyMobileS" because we can get this size as `size.laptop - 1`

Reply:
use next code: onlyMobileS: `(max-width: calc(${size.laptop}px - 0.02px))`,